webfront/views/modifiers.py

```diff
@@ -457,7 +487,7 @@ def get_entry_annotation(field, general_handler):
 def get_entry_annotation_info(field, general_handler):
     queryset = general_handler.queryset_manager.get_queryset()
     for entry in queryset:
-        data = entry.entryannotation_set.filter(type=field).values(
+        data = entry.entryannotation_set.filter(go=field).values(
```

Comment:
```suggestion
        data = entry.entryannotation_set.filter(type=field).values(
```

This is likely to be a copy/paste that went wrong :wink: 


This method is used by the `annotation:info` modifier to retrieve the metadata of an entry annotation without the annotation itself.

For instance, [https://www.ebi.ac.uk/interpro/api/entry/pfam/PF00003?annotation=alignment:seed](https://www.ebi.ac.uk/interpro/api/entry/pfam/PF00003?annotation=alignment:seed) returns the (gzipped) seed alignment:

```sh
$ curl -s 'https://www.ebi.ac.uk/interpro/api/entry/pfam/PF00003?annotation=alignment:seed' | gzip -cd | head
# STOCKHOLM 1.0
#=GF ID   7tm_3
#=GF AC   PF00003.27
#=GF DE   7 transmembrane sweet-taste receptor of 3 GCPR
#=GF AU   Sonnhammer ELL;0000-0002-9015-5588
#=GF SE   Prosite
#=GF GA   33.50 33.50;
#=GF TC   33.50 33.50;
#=GF NC   33.40 33.40;
#=GF BM   hmmbuild HMM.ann SEED.ann
```

And https://www.ebi.ac.uk/interpro/api/entry/pfam/PF00003?annotation:info=alignment:seed returns the metadata of the seed alignment, but without the alignment itself:

```sh
$ curl -s 'https://www.ebi.ac.uk/interpro/api/entry/pfam/PF00003?annotation:info=alignment:seed' | python -m json.tool
{
    "accession": "PF00003",
    "type": "alignment:seed",
    "mime_type": "application/gzip",
    "num_sequences": 538
}
```